django-stubs/forms/renderers.pyi

```diff
@@ -30,6 +31,7 @@ class Jinja2(EngineMixin, BaseRenderer):
 class Jinja2DivFormRenderer(Jinja2):
     form_template_name: str
     formset_template_name: str
+    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
```

Comment:
```suggestion
    def __init__(self) -> None: ...
```

https://github.com/django/django/blob/e72049aa6302411d8cdf2e798e15fa38e76b92fc/django/forms/renderers.py#L94-L100

This code calls `super().__init__(*args, **kwargs)`, but no parents seem to define any custom `__init__` methods.

So, it is `object.__init__`. Maybe we can even ignore it in `allowlist.txt`

Reply:
I understand :) I'll add that to the `allowlist.txt`, thank you!